2.2/.sti/bin/run

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+function is_puma_installed() {
+  [ ! -f Gemfile.lock ] && return 1
+  grep ' puma ' Gemfile.lock >/dev/null
+}
+
+set -e
+
+export RACK_ENV=${RACK_ENV:-"production"}
+export RAILS_ENV=${RAILS_ENV:-"${RACK_ENV}"}
```

Comment:
I've we are going to split the ruby and ror images we dont need this to be set here, but in the ror image